ddtrace/contrib/pymongo/client.py

```diff
@@ -116,11 +116,12 @@ def send_message_with_response(self, operation, *args, **kwargs):
                 span_type=mongox.TYPE,
                 service=pin.service) as span:
 
-            span.resource = _resource_from_cmd(cmd)
             span.set_tag(mongox.DB, cmd.db)
             span.set_tag(mongox.COLLECTION, cmd.coll)
             span.set_tags(cmd.tags)
 
+            _set_query_metadata(span, cmd)
```

Comment:
We might want a comment here that says this function will set the `span.resource` and add a `mongodb.query` tag if needed.